packages/bot-web-ui/src/components/network-toast-popup/index.ts

```diff
@@ -0,0 +1,3 @@
+import NetworkToastPopup from './network-toast-popup';
+
+export default NetworkToastPopup;
```

Comment:
@shafin-deriv add a empty line after the export statement as per the standards 🙏